javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,10 @@
+//Ez Namey
+
+const firstWordsArray = ['Easy', 'Corporate', 'Awesome', 'Tech', 'Business', 'Creative', 'Data', 'Alpha', 'Delta', 'Cloud'];
+const secondWordsArray = ['Beta', 'Infinite', 'Level', 'Collective', 'Collaboration', 'Server', 'Craft', 'Brand', 'Vital', 'Blink'];
+
+const randomNumber = Math.floor(Math.random() * 10) + 0;
+const startUpName = firstWordsArray[randomNumber] + ' ' + secondWordsArray[randomNumber];
+const numberOfCharacters = startUpName.length;
+
+console.log('The startup is: ' + startUpName + ' contains ' + numberOfCharacters + ' Characters');
```

Comment:
You should consider using [backticks for your string interpolation](https://dmitripavlutin.com/string-interpolation-in-javascript/). It saves you a bunch of quotation marks and pluses:
```javascript
console.log(`The startup is: ${startUpName} contains ${numberOfCharacters} Characters`); 
```

Reply:
Yeah-this is much easier. I did change all my files using this now